.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The `es2024` environment is specified, but ECMAScript version in `parserOptions` is set to 12. Consider aligning these settings to avoid potential inconsistencies.